packages/components/src/components/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,57 @@
+@use "@/styles/mixins/focus";
+
+.accordion {
+  display: flex;
+  flex-direction: column;
+
+  /* Elements */
+
+  .header {
+    display: flex;
+  }
+
+  .headerButton {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-align: start;
+    flex-grow: 1;
+    padding-inline: var(--accordion--header-padding-x);
+    padding-block: var(--accordion--header-padding-y);
+    border-radius: var(--accordion--header-corner-radius);
+    column-gap: var(--accordion--heading-to-chevron-spacing);
+    border: none;
+    background: none;
+
+    @include focus.focus;
+
+    &:hover {
+      background-color: var(--accordion--header-background-color--hover);
+    }
+  }
+
+  .content {
+    display: grid;
+    grid-template-rows: 0fr;
+    transition-property: grid-template-rows, visibility, padding;
+    transition-duration: var(--transition--duration--slow);
+    padding-inline: var(--accordion--content-padding);
+    visibility: hidden;
+  }
+
+  .contentInner {
+    overflow: hidden;
+  }
+
+  &.expanded {
+    .content {
+      grid-template-rows: 1fr;
+      padding-block: var(--accordion--content-padding);
+      visibility: visible;
+    }
+
+    .chevron {
+      transform: rotate(180deg);
```

Comment:
In Verwendung mit dem Label ist das Icon grau. Ist das bewusst?

![Bildschirmfoto 2024-05-03 um 08 02 20](https://github.com/mittwald/flow/assets/4696750/90c68682-6161-402f-b4f6-b72fc0ad1b2d)

Reply:
jupp, das ist so gewollt